plugins/janus_videoroom.c

```diff
@@ -2321,6 +2330,43 @@ void janus_videoroom_hangup_media(janus_plugin_session *handle) {
 	}
 }
 
+static void janus_videoroom_sdp_v_format_vp8(char *mline, int mline_size, int pt, int b, const char *video_mode) {
```

Comment:
Since all these methods have the same parameters, wouldn't it be simpler to have a `janus_videoroom_sdp_v_format` method that also has the codec as one of the arguments, and then do a switch in there to apply the correct format? We already have enums for the purpose: https://github.com/meetecho/janus-gateway/blob/master/plugins/janus_videoroom.c#L346